src/components/experiments/single-view/results/MetricAssignmentResults.tsx

```diff
@@ -127,6 +127,36 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 )
 
+type StringifiedStatisticalDifference = 'true' | 'false'
+
+// Practical Difference Status -> (string) Statistical Difference -> string
+const differenceOverviewMessages: Record<
+  Analyses.PracticalSignificanceStatus,
+  Record<StringifiedStatisticalDifference, string>
+> = {
+  [Analyses.PracticalSignificanceStatus.Yes]: {
+    true: `There is high certainty that the change is pratically significant`,
+    false: `There is high certainty that the change is pratically significant`,
```

Comment:
Typo + add a period for consistency.

```suggestion
    true: `There is high certainty that the change is practically significant.`,
    false: `There is high certainty that the change is practically significant.`,
```